js/apps/admin-ui/src/realm-settings/user-profile/UnmanagedAttributesTab.tsx

```diff
@@ -0,0 +1,141 @@
+import { useTranslation } from "react-i18next";
+import { KeycloakSpinner } from "../../components/keycloak-spinner/KeycloakSpinner";
+import { useUserProfile } from "./UserProfileContext";
+import {
+  FormGroup,
+  Grid,
+  GridItem,
+  PageSection,
+  Switch,
+} from "@patternfly/react-core";
+import { HelpItem } from "ui-shared";
+import { UnmanagedAttributePolicy } from "@keycloak/keycloak-admin-client/lib/defs/userProfileMetadata";
+import { FormAccess } from "../../components/form/FormAccess";
+import { useEffect, useState } from "react";
+
+export const UnmanagedAttributesTab = () => {
+  const { config, save } = useUserProfile();
+  const { t } = useTranslation();
+  const [enableUnmanaged, setEnableUnmanaged] = useState(false);
+
+  useEffect(() => {
+    setEnableUnmanaged(config?.unmanagedAttributePolicy != undefined);
+  }, [config]);
+
+  if (!config) {
+    return <KeycloakSpinner />;
+  }
+
+  function setPolicy(policy?: UnmanagedAttributePolicy) {
```

Comment:
This doesn't really 'set' the policy as much as it saves it. Perhaps `savePolicy()` would be a better name?
```suggestion
  function savePolicy(policy?: UnmanagedAttributePolicy) {
```

Reply:
We no longer have this tab. Please, see one of the comments above about how we are updating the state now.